stylesheets/mobile/stylesheet-blog-mobile.css

```diff
@@ -0,0 +1,132 @@
+* {
```

Comment:
While its good to break up your css styles, have duplicates across different sheets can make it difficult to update your styles, since you might make a change on one sheet but forget to update the others. Better practice is to have one main stylesheet for common styles unique to all pages. (Think the header, footer, font family etc. )Then have seperate sheets like you have done, but only have styles for elements unique to that page, like the form or blog. then load multiple sheets. Remember that CSS loads top down and whatever is loaded last will overwrite whatever styles were loaded before.